pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -919,6 +920,18 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 		}
 	}
 
+	if spec.Domain.Devices.BlockMultiQueue != nil {
```

Comment:
Seems like we'd need to actually check the value here if it is != nil.  This is a boolean right?

Reply:
Thanks @davidvossel. I think you're right. setting BlockMultiQueue explicitly to false would erroneously enforce the check for CPU requests.